figures/discovery/monolith-discoverers.mmd

```diff
@@ -13,6 +13,7 @@ classDiagram
     DnsDiscoveryConfig : +u16 monolith_port
     DnsDiscoveryConfig : +Option~String~ dns_server
     DnsDiscoveryConfig : +String query
+    DnsDiscoveryConfig : +Int PollingDuration
```

Comment:
These should be optional, and should be a real rust number type (some unsigned int) rather than `Int`.

nit: should also be snake_case. naming must match what you put in the code